src/pages/Home/index.js

```diff
@@ -62,7 +63,7 @@ const randomNumber = (min, max) => {
 
 const randomNumberList = data => {
```

Comment:
logic with cut product and choose random 9 product from topList move to `action`